2020F-usfca-btec-640.github.io/404.html

```diff
@@ -0,0 +1,25 @@
+---
+permalink: /404.html
```

Comment:
This file should not be a part of the PR. Please remove it so we don't have to deal with merge conflicts.